examples/TestNetCoreConsole/NamedPipeSignaler.cs

```diff
@@ -238,12 +248,14 @@ private void ProcessIncomingMessages()
         /// Entry point for the writing task dequeuing outgoing messages and
         /// writing them to the sending pipe.
         /// </summary>
-        private void WriteOutgoingMessages()
+        private async Task WriteOutgoingMessagesAsync(CancellationToken ct)
         {
-            // GetConsumingEnumerable() will block when no message is available,
-            // until CompleteAdding() is called from Stop().
-            foreach (var msg in _outgoingMessages.GetConsumingEnumerable())
+            while (await _outgoingMessages.OutputAvailableAsync(ct))
             {
+                // GetConsumingEnumerable() will wait until messages are available when no message is available,
```

Comment:
Outdated comment? `GetConsumingEnumerable()` was removed, no?

Reply:
Good catch! fixed in 64f463e65cf4ab4e6294e35fce2b2f0ee4602d44